build.gradle

```diff
@@ -8,11 +8,20 @@ apply plugin: 'java'
 
 apply plugin: 'idea'
 apply plugin: 'eclipse'
+apply plugin: 'jacoco'
 
 ext.versions = [
     battlecode: new File(projectDir, "version.txt").text.trim()
     //scala: '2.11.7'
 ]
+jacocoTestReport {
```

Comment:
this section is jacoco's configuration for our project. Although I don't think html.destination even works. I basically punched a configuration file for hours with help from stackoverflow until it worked.